### ISSUE TITLE
Locale file not load when navigate from non-default to default language

### ISSUE THREAD

**Issue by Bigdragon13th** [nuxt-modules/i18n#843]
### Version
nuxt-i18n: 6.13.7
nuxt: 2.14.1

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [x] universal
 - [] spa

### Nuxt-i18n configuration
```
i18n: {
    locales: [
      {
        code: "en",
        name: "EN",
        file: "en.json"
      },
      {
        code: "th",
        name: "TH",
        file: "th.json"
      }
    ],
    defaultLocale: "en",
    strategy: "prefix",
    lazy: true,
    langDir: "locales/",
    detectBrowserLanguage: false
  }
```

### Reproduction Link
https://codesandbox.io/s/friendly-lamport-326c8

### Steps to reproduce
- Open the sandbox link
- Start with `/th` link
- Click 'en'
- The locale for `en` is not loaded

### What is Expected?
- The locale for `en` should load

### What is actually happening?
- The locale for `en` is not loaded

### Etc.
- If you start with `/en` (the default locale) and click `th`, the locale is working properly
- nuxt-i18n version 6.13.6 not having this issue

**Comment by rchl** [comment]
Silly mistake, sorry about that.
It would have worked if you would set `i18n.fallbackLocale` to `en` also but maybe you don't want that and it shouldn't be necessary anyway.

**Comment by rchl** [comment]
Released in [v6.13.9](https://github.com/nuxt-community/i18n-module/releases/tag/v6.13.9)

**Comment by Bigdragon13th** [comment]
It's working. Thank you for very quick fix @rchl !
